UI_UX/Frontend/src/components/AppHeader.js

```diff
@@ -1,12 +1,29 @@
 import React from 'react';
-import logo from '../images/mybit-logo-white.png';
+import logo from '../images/mybit-logo-white.svg';
 import '../styles/AppHeader.css';
+import { Logo } from './Logo';
+import { ExchangeRate } from './ExchangeRate';
+import { AccountInfo } from './AccountInfo';
 
-export const AppHeader = () => {
+export const AppHeader = ({
+  loadingBalance = false,
+  exchangeRate = 2.23,
+  myBitBalance = 215,
+  loadingAccountInfo = false,
+  ethBalance = 20,
+  address = '0xde0BF ...'
```

Comment:
Ellipsis should be added in CSS, not hardcoded. Use a full address example and then add the proper CSS rules to show ellipsis.

Reply:
fixed